lib/streamlit/elements/chat.py

```diff
@@ -160,6 +195,112 @@ def chat_message(
 
         return self.dg._block(block_proto=block_proto)
 
+    @gather_metrics("chat_input")
+    def chat_input(
+        self,
+        placeholder: str = "Your message",
+        *,
+        key: Key | None = None,
+        max_chars: int | None = None,
+        disabled: bool = False,
+        on_submit: WidgetCallback | None = None,
+        args: WidgetArgs | None = None,
+        kwargs: WidgetKwargs | None = None,
+    ) -> str | None:
+        r"""Display a chat input widget.
+
+        Parameters
+        ----------
+        placeholder : str
+            A placeholder text shown when the chat input is empty. Defaults to
+            "Your message". For accessibility reasons, you should not use an
+            empty string.
+        max_chars : int or None
+            The maximum number of characters that can be entered. If None
+            (default), there will be no maximum. Can be supplied by keyword only.
```

Comment:
I think we can either remove the `Can be supplied by keyword only` (my preference) or add it to the other params below as well. Same also with `disabled`.